IABot/checkIfDead.php

```diff
@@ -58,23 +58,22 @@ public function checkDeadlinks( $urls ) {
 			curl_setopt( $curl_instances[$id], CURLOPT_NOBODY, 1 );
 			curl_setopt( $curl_instances[$id], CURLOPT_RETURNTRANSFER, true );
 			curl_setopt( $curl_instances[$id], CURLOPT_FOLLOWLOCATION, true );
-			curl_setopt( $curl_instances[$id], CURLOPT_TIMEOUT, 30 ); // Set 30 seconds timeout for the entire curl operation to take place
+			curl_setopt( $curl_instances[$id], CURLOPT_TIMEOUT, 60 ); // Set 60 second timeout for the entire curl operation to take place
```

Comment:
In other words, any URL that failed the HEAD request and needs a GET request, will be placed in a separate array.  That array is then passed to then passed to itself in a recursive call with an additional $full variable set to true for example.  Seems much simpler.